TACC/Core-Portal
# Quick: Resolve Secrets Delta

## Overview: ##

- [X] Attempt to make the `settings_secrets.example.py` consistent with [Frontera Stache secrets](https://stache.utexas.edu/entry/82d46ac7e6c1d4062352e4be6201ed9d).\*

## Related Jira tickets: ##

* N/A

## Summary of Changes: ##

- __Fix__: Resolve discrepancies between order of values in repo example secrets and Frontera Stached secrets.
- __Fix__: Add missing settings to repo example secrets from Frontera Stached secrets.

## Testing Steps: ##
1. Create a fresh repo clone.
2. Follow `README.md` steps.
3. Set up a fresh dev environment.
4. Ensure "stuff works".

## UI Photos:

…

## Notes: ##
